travis_configure.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
```

Comment:
Usually `sh` is used for package configuration/installation. Is there any special reason why `bash` is needed?